region/maridia/outer/Fish Tank.json

```diff
@@ -896,15 +924,21 @@
       },
       "requires": [
         "canWaterShineCharge",
+        "canTrickyJump",
+        "h_canShineChargeMaxRunway",
         {"or": [
           {"shinespark": {"frames": 41, "excessFrames": 7}},
           {"and": [
+            "canDodgeWhileShooting",
             "canMidairShinespark",
             {"shinespark": {"frames": 38, "excessFrames": 7}}
           ]}
         ]}
       ],
-      "note": "Start the dash as close to the door transition as possible in order to have a larger window for storing the shinecharge."
+      "note": [
+        "Start the dash as close to the door transition as possible in order to store the shinespark in time.",
+        "Stand 2 to 5 pixels from the door for run speed to be low enough going into the next room."
```

Comment:
In my testing, only 2 or 3 pixels from the door work reliably. 4 pixels away can work but it depends on subpixels: you must be on the leftmost 75% of the pixel in that case. This is assume holding dash the whole time. By pressing dash after moving, it can work further away but then there is a risk of being late on the dash press so it doesn't necessarily seem to help unless you have a lot of extra runway, which falls under the other strat.